### ISSUE TITLE
'Home' and 'Back' gestures aren't working

### ISSUE THREAD

**Issue by iamsonusaroy** [KieronQuinn/TapTap#102]
Everything's working perfectly except 'home' and 'back' gestures on Realme 2 Pro. I'm on Havoc OS (Android 10). And using lawnchair launcher, if that helps.

**Comment by KieronQuinn** [comment]
Those options call the built in system API for accessibility. There's no alternative to doing that without root, which I'm not prepared to implement for the few people with this bug
